docs/Formula-Cookbook.md

```diff
@@ -639,7 +639,7 @@ Patches can also be embedded by passing a string. This makes it possible to prov
 patch :p0, "..."
 ```
 
-In embedded patches, the string "HOMEBREW\_PREFIX" is replaced with the value of the constant `HOMEBREW_PREFIX` before the patch is applied.
+In patches, the string "HOMEBREW\_PREFIX" is replaced with the value of the constant `HOMEBREW_PREFIX` before the patch is applied.
```

Comment:
Just a note that this may need updated based on what we decide above

Reply:
Done in ff03947ba0aab28c8fadfef70d34c6b6f49408ca